src/hayai_console_outputter.hpp

```diff
@@ -207,37 +217,62 @@ namespace hayai
                           result.RunsPerSecondMinimum(),
                           result.RunsPerSecondAverage(),
                           "runs/s");
+            PAD("Median performance: "
+		<< result.RunsPerSecondMedian() << " runs/s "
+		<< "(" << Console::TextCyan << " "
+		<< result.RunsPerSecondQuartile1() << " | "
+		<< result.RunsPerSecondQuartile3() << " "
+		<< Console::TextDefault << ")"
+		);
 
+            PAD("");
             _stream << Console::TextBlue << "[ITERATIONS] "
                     << Console::TextDefault
                     << std::setprecision(3)
                     << "       Average time: "
-                    << result.IterationTimeAverage() / 1000.0 << " us"
+                    << result.IterationTimeAverage() / 1000.0 << " us "
+		    << "(" << Console::TextBlue << "~"
+		    << result.IterationTimeStdDev() / 1000.0 << " us"
+		    << Console::TextDefault << ")"
                     << std::endl;
 
-            PAD_DEVIATION_INVERSE("Fastest: ",
+            PAD_DEVIATION_INVERSE("Fastest time: ",
                                   (result.IterationTimeMinimum() / 1000.0),
                                   (result.IterationTimeAverage() / 1000.0),
                                   "us");
-            PAD_DEVIATION_INVERSE("Slowest: ",
+            PAD_DEVIATION_INVERSE("Slowest time: ",
                                   (result.IterationTimeMaximum() / 1000.0),
                                   (result.IterationTimeAverage() / 1000.0),
                                   "us");
+            PAD("Median time: "
+		<< result.IterationTimeMedian() / 1000.0 << " us "
+		<< "(" << Console::TextCyan << " "
+		<< result.IterationTimeQuartile1() / 1000.0 << " | "
+		<< result.IterationTimeQuartile3() / 1000.0 << " "
+		<< Console::TextDefault << ")"
+		);
 
             _stream << std::setprecision(5);
 
             PAD("");
             PAD("Average performance: " <<
                 result.IterationsPerSecondAverage() <<
-                " iterations/s");
+                " iter/s " );
```

Comment:
Yes you are right again, this change is unwanted as well. 
Regarding the rewording, I've forgotten to change it back, sorry.